docs/modules/ROOT/pages/overview.adoc

```diff
@@ -14,6 +14,18 @@ See xref:install-clc.adoc[installation documentation].
 
 Example use cases for the Hazelcast CLC.
 
+=== Run SQL Queries
+
+Run xref:clc-sql.adoc[SQL] queries against your cluster and display the output as a table or in a variety of text formats, such as JSON and CSV.
+
+=== Manage Viridian Clusters
+
+Using CLC, you can xref:clc-viridian.adoc[create and manage] Viridian clusters and custom classes on those clusters.
+
+=== Create Data Pipelines
+
+You can xref:clc-job.adoc[create and manage] data pipelines using CLC. Check out the Platform documentation for more information about data pipelines.
```

Comment:
```suggestion
You can xref:clc-job.adoc[create and manage] data pipelines using the Hazelcast CLC. Check out the Platform documentation for more information about data pipelines.
```
It would be good to provide a link to the appropriate page.

Reply:
Here is the syntax xref:hazelcast:pipelines:overview.adoc[Platform documentation]

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/241/commits/078ec910da7d1053d167e6eafce0feadee1af0d1